src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -816,6 +817,8 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.is_stale = True
 
     def _applicable(self):
+        if self.command_loader.skip_applicability:
+            return True
```

Comment:
What's this for?

Reply:
Under normal operations, if you reach "load_arguments" it mean you specified a specific command. So the argument_context object will check if its scope is relevant to the command you are running. If it isn't, the entire block is skipped. However, for interactive, when you want to load the world, you don't want it to do that optimization, so interactive sets this flag so that it is forced to load all argument contexts.